lib/postgrex/protocol.ex

```diff
@@ -654,18 +675,28 @@ defmodule Postgrex.Protocol do
     end
   end
 
+  defp execute_send( s, status, %Stream{ state: :suspended, portal: portal, max_rows: max_rows, query: query } = stream, _params, buffer ) do
+    %Query{ name: name } = query
+    messages = [ msg_execute(name_port: portal, max_rows: max_rows) | sync_msgs( status, name ) ]
+    send_and_recv( s, status, stream, buffer, messages, &execute_recv/4 )
```

Comment:
Elsewhere we use `status` to determine flow instead of funs, it would be nice to be consistent throughout.